mailchimptemplate.php

```diff
@@ -179,4 +115,42 @@ function mailchimptemplate_civicrm_navigationMenu(&$params)
             'active' => 1,
         ],
     ];
+
+    // Settings menu
+    _mailchimptemplate_civix_insert_navigation_menu($params, 'Administer/CiviMail', [
+        'label' => E::ts('MailChimp Settings'),
+        'name' => 'MailChimp Settings',
+        'url' => 'civicrm/mailchimptemplate/settings',
+        'permission' => 'administer CiviCRM',
+        'operator' => 'OR',
+        'separator' => 0,
+        'active' => 1
+    ]);
+
+    _mailchimptemplate_civix_navigationMenu($params);
+}
+
+function mailchimptemplate_civicrm_xmlMenu(&$files)
+{
+    $files[] = dirname(__FILE__) . '/xml/Menu/mailchimptemplate.xml';
+}
+
+/**
+ * Implements hook_civicrm_postInstall().
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_postInstall
+ */
+function mailchimptemplate_civicrm_postInstall()
```

Comment:
Why do we need to implement this hook?

Reply:
We don't need this hook implementation. Civix autogenerated it. If I will delete, civix will regenerate it next time.

Reply:
OK for now, it will be removed at some point in the future anyway